test/behavior/tendrl/login.feature

```diff
@@ -0,0 +1,45 @@
+Feature: LOGIN
+
+  Scenario: Type username then delete
+    Given I open the Tendrl url
+    When  I type something in the username field
+    And   I delete what is in the username field
+    Then  The text "username is required" should appear under the username box
+
+  Scenario: Type password then delete
+    Given I open the Tendrl url
+    When  I type something in the password field
+    And   I delete what is in the password field
+    Then  The text "password is required" should appear under the password box
+
+  Scenario: Click login with empty fields
+    Given I open the Tendrl url
+    When  I click the Login button
+    Then  The “Please fill out this field" popup should appear under the username box
+
+  Scenario: Click login with username but no password
+    Given I open the Tendrl url
+    When  I type something in the username field
+    And   I click the Login button
+    Then  The “Please fill out this field" popup should appear under the password box
+
+  Scenario: Click login with password but no username
+    Given I open the Tendrl url
+    When  I type something in the password field
+    And   I click the Login button
+    Then  The “Please fill out this field" popup should appear under the username box
+
+  Scenario: Incorrect Username and Password
+    Given I open the Tendrl url
+    When  I type in an incorrect username
+    And   I type in an incorrect password
+    And   I click the Login button
+    Then  The “username or password does not match” should appear above the username box
+    And   The text "password is required" should appear under the password box
+
+  Scenario: Correct Username and Password
```

Comment:
same https://github.com/Tendrl/ui/blob/master/src/modules/login/login.js#L49

Reply:
What change should I be making here?

Reply:
Sorry I think I have pasted some other link